src/components/Job.jsx

```diff
@@ -0,0 +1,77 @@
+'use client';
```

Comment:
@userimack can we use existing card. designs if possible, below are some examples,
- https://2025.pyconfhyd.org/speakers
- https://2025.pyconfhyd.org/our-team

Reply:
Done, I have used the our-team's design and updated the PR
Please check